gateways/clients/hathor_core_client.py

```diff
@@ -33,14 +33,15 @@
 class HathorCoreAsyncClient:
     DEFAULT_TIMEOUT = 60  # seconds
 
-    def __init__(self, domain: Optional[str] = None) -> None:
+    def __init__(self, url: Optional[str] = None, prefix: str = "https://") -> None:
         """Client to make async requests
 
-        :param domain: domain where the requests will be made, defaults to config `hathor_core_domain`
-        :type domain: str, optional
+        :param url: url where the requests will be made, defaults to config `hathor_core_url`
+        :type url: str, optional
```

Comment:
suggestion: Add the `prefix` parameter to the docstrings

Reply:
Actually this is a leftover, I've removed it: https://github.com/HathorNetwork/hathor-explorer-service/pull/326/commits/71588c0930c6dae82d77c50a2e24e7552ff3edcd